.github/workflows/formatting.yml

```diff
@@ -32,10 +32,10 @@ jobs:
           - 'larpandoradlcontent'
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       - name: Check clang-format
-        uses: jidicula/clang-format-action@v4.10.2
+        uses: jidicula/clang-format-action@v4.13.0
         with:
-          clang-format-version: '13'
+          clang-format-version: '18'
```

Comment:
This isn't actually actively used...if we want to, we could get the whole code base under `clang-format` compliance, then enable this on PRs.